tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py

```diff
@@ -0,0 +1,144 @@
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - numeric
+      # BatchRequest yielding exactly one batch (March, 2019 trip data)
+      batch_request:
+        datasource_name: taxi_pandas
+        data_connector_name: monthly
+        data_asset_name: my_reports
+        data_connector_query:
+          index: -1
+    parameter_builders:
+      - parameter_name: min_range
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request:
+            datasource_name: taxi_pandas
+            data_connector_name: monthly
+            data_asset_name: my_reports
+            data_connector_query:
+              index: "-4:-1"
+        metric_name: column.min
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
+      - parameter_name: max_range
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request:
+            datasource_name: taxi_pandas
+            data_connector_name: monthly
+            data_asset_name: my_reports
+            data_connector_query:
+              index: "-4:-1"
+        metric_name: column.max
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
+    expectation_configuration_builders:
+      - expectation_type: expect_column_min_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
+        column: $domain.domain_kwargs.column
+        min_value: $parameter.min_range.value.min_value
+        max_value: $parameter.min_range.value.max_value
+        mostly: $variables.mostly
+        meta:
+          profiler_details: $parameter.min_range.details
+      - expectation_type: expect_column_max_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
+        column: $domain.domain_kwargs.column
+        min_value: $parameter.max_range.value.min_value
+        max_value: $parameter.max_range.value.max_value
+        mostly: $variables.mostly
+        meta:
+          profiler_details: $parameter.max_range.details
+"""
+
+data_context = DataContext()
+
+
+# Instantiate Profiler
+full_profiler_config_dict: dict = yaml.load(profiler_config)
+rules_configs: dict = full_profiler_config_dict.get("rules")
+variables_configs: dict = full_profiler_config_dict.get("variables")
+
+profiler: Profiler = Profiler(
+    rules_configs=rules_configs,
+    variables_configs=variables_configs,
+    data_context=data_context,
+)
+
+suite = profiler.profile()
+print(suite)
+
+
+"""
+{
+  "meta": {
+    "great_expectations_version": "0.13.19+58.gf8a650720.dirty"
+  },
+  "data_asset_type": null,
+  "expectations": [
+    {
+      "kwargs": {
```

Comment:
@talagluck I am not sure if these min and max values are correct.  They could be completely correct -- we just need to discuss briefly.  Please see my comment at the end.  Thanks.

Reply:
@alexsherstinsky - this is the output from running the integration test script. Happy to discuss if it should be something else.